_spinnaker/pacrd.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: PaCRD
+order: 171
+---
+
+<!-- Note:
+  Setting Order as 171 so it shows up at the bottom on the Spinnaker section
+  of the Armory docs, given it's experimental status. When/if this product
+  hits GA we probably want to move it further up the list and/or replace
+  Dinghy docs in time.
+!-->
+
+PaCRD is considered [**Experimental**] software. This feature is working and
+installable but is missing functionality and subject to rapid change.
+
+__Interested in joining this experiment?
```

Comment:
This is how Akshay wishes to be contacted for folks joining the experiment.

Reply:
I was thinking more about this - lets use pipelines-as-code@armory.io